search-commons/src/main/java/no/unit/nva/search/resource/ContributorNodeFilter.java

```diff
@@ -0,0 +1,67 @@
+package no.unit.nva.search.resource;
+
+import static no.unit.nva.constants.Words.AFFILIATIONS;
+import static no.unit.nva.constants.Words.COUNTRY_CODE;
+import static no.unit.nva.constants.Words.ENTITY_DESCRIPTION_CONTRIBUTORS_PATH;
+import static no.unit.nva.constants.Words.IDENTITY;
+import static no.unit.nva.constants.Words.NO;
+import static no.unit.nva.constants.Words.VERIFICATION_STATUS;
+import static no.unit.nva.constants.Words.VERIFIED;
+
+import static nva.commons.core.attempt.Try.attempt;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+
+import no.unit.nva.search.common.records.Midas;
+
+import nva.commons.core.JacocoGenerated;
+
+public class ContributorNodeFilter implements Midas {
+
+    // (package private)
```

Comment:
Is this comment necesary?

Reply:
I have had question about this on qa's before, so I added it, to ensure there where no confusion about it's intentions

So, I wish it wasn't.